xilem_web/src/svg/common_attrs.rs

```diff
@@ -67,10 +67,11 @@ fn join(iter: &mut impl Iterator<Item: std::fmt::Display>, sep: &str) -> String
 fn brush_to_string(brush: &Brush) -> String {
     match brush {
         Brush::Solid(color) => {
-            if color.a == 0 {
+            let [r, g, b, a] = color.to_rgba8().to_u8_array();
+            if a == 0 {
                 "none".into()
             } else {
-                format!("#{:02x}{:02x}{:02x}", color.r, color.g, color.b)
+                format!("#{:02x}{:02x}{:02x}", r, g, b)
```

Comment:
@Philipp-M Was it intentional here to drop the alpha channel?

If not, then we can use the new `LowerHex` formatter for `Color`....

Reply:
I think this is originally from Raph, but IIRC this is indeed intentional, see also [here](https://stackoverflow.com/questions/6042550/svg-fill-color-transparency-alpha), the alpha channel gets separately applied with the `fill-opacity` and `stroke-opacity` attribute

Reply:
I've changed this code in the new version now and added a test for the behavior to make sure it is doing the right thing.